frontend/package.json

```diff
@@ -119,6 +120,7 @@
       "jest-css-modules-transform": "^4.4.2",
       "jest-environment-jsdom": "28.1.2",
       "jest-watch-typeahead": "1.1.0",
+      "msw": "^0.49.2",
```

Comment:
Do we want to include `^` or just use a specific version?

Reply:
I'll make the change since that's what we kind of do for the rest of the packages.

Reply:
Addressed in [ef53bd7](https://github.com/iFixit/react-commerce/pull/1216/commits/ef53bd774d784baf7cd5d0479c13e83571b08e60)